src/components/accessories/admin/types/components/agetypes/ageTypesForm/AgeTypeForm.tsx

```diff
@@ -0,0 +1,42 @@
+import TextField from "components/accessories/textField/TextField";
+import React, { FC } from "react";
+import { IAgeTypeFormProps } from "./types";
+
+const AgeTypeForm: FC<IAgeTypeFormProps> = ({
+  formik,
+  getErrorText,
+  isValid,
+  t,
+  index,
+}) => {
```

Comment:
```suggestion
const AgeTypeField: FC<IAgeTypeFieldProps> = ({
  field,
  getErrorText,
  isValid,
  onBlur
}) => {
    const { t } = useTranslation()
```

Reply:
In my opinion the best way here is to pass `formik` object so that sub-components could just use props of formik they need. Splitting `formik` into possible props that could be used by sub-components is unnecessary as the actual implementation isn't a bad practice and doesn't cause any performance issue. Splitting `formik` just complicates things for nothing.